src/features/auth/actions.ts

```diff
@@ -42,6 +43,7 @@ export const loadUserAsync = () => async (dispatch: ThunkDispatchAny) => {
     dispatch(loadUser.request());
     const res = await axios.get('/auth');
     dispatch(loadUser.success(res.data));
+    dispatch(resetContent());
```

Comment:
To avoid unnecessarily reloading content when a user already has a valid token, I would suggest only resetting the content when the user's authorization status changes (login, logout, register). See my separate PR, based on yours.